### ISSUE TITLE
[Question] How add a customized Title Graphic to the Book

### ISSUE THREAD

**Issue by Dream-Master** [Drethic/QuestBook#9]
I try to find some info’s here but all info’s was given in the wiki and book not working for me.
For testing I put the graphic inside the jar file but I guess there is an easier way.
If I put the graphic inside the better questing Folder in resources Folder how the path have to look like in the config file?

Thanks in advance.

**Comment by Drethic** [comment]
Are you trying to replace the book image, or the image that loads when you open the quest GUI?

**Comment by Dream-Master** [comment]
When I open the Book(Quest Gui). The title Page.

**Comment by Drethic** [comment]
If you go into config/betterquesting.cfg and edit this line:

```
# Texture used in the quest tile card [default: betterquesting:textures/gui/default_title.png]
S:"Title Texture"=betterquesting:textures/gui/default_title.png
```

Change default_title.png to the name of your png file.  Here is the source image so you can get the dimensions for yours.

https://github.com/Funwayguy/BetterQuesting/blob/master/src/main/resources/assets/betterquesting/textures/gui/default_title.png

Once you have the config updated and your image created, I would recommend getting the Resource Loader Mod (http://minecraft.curseforge.com/projects/resource-loader).  Then in the root directory create resources/betterquesting/textures/gui and place your image in there.  It should get picked up in game.

**Comment by Drethic** [comment]
Looks like @Funwayguy has an easier way to do it:

https://github.com/Funwayguy/BetterQuesting/wiki/Resources

You just put the image in config/betterquesting/resources/< pack name>/textures/gui and in the config type in < pack name>:textures/gui/< image name>.png

**Comment by Dream-Master** [comment]
Ah ok thanks will try this.

**Comment by Dream-Master** [comment]
I try it and get this. What i do wrong ?

![image](https://cloud.githubusercontent.com/assets/2970688/14967505/01d36b78-10b8-11e6-85b6-3833dd25e8cf.png)

**Comment by Drethic** [comment]
I was able to reproduce what you are seeing.
In the config change your "Title Texture" to this:

```
gtnh:textures/gui/gtnh_title.png
```

In the config folder change all GTNH to gtnh.  For some reason Better Questing is getting the information as all lower case which doesn't exist.

**Comment by Dream-Master** [comment]
Will change the packname name to "gtnh" and test it again.

**Comment by Dream-Master** [comment]
Ok works with "gtnh". I let it open so long time it is fixed ok ?

**Comment by Drethic** [comment]
If you look at Funwayguy/BetterQuesting#27 you'll see a note from @Funwayguy saying it's bad practice to use anything but lower case for resources.  He's looking at implementing it, but I think it would be better to follow the standards and keep it all lower case.

**Comment by Dream-Master** [comment]
Sure but more people will run into it and so i think its better it is supported.

**Comment by Drethic** [comment]
@Funwayguy has updated the Wiki to warn about using all lowercase for the image path and config which closes out Funwayguy/BetterQuesting#27 and this issue.
